### PR TITLE
Filter based on GCN properties

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR enables filtering based on GCN properties.

### CODE REVIEW

File: skyportal/models/observation_plan.py [hunk 1]
```diff
@@ -58,6 +58,11 @@ class DefaultObservationPlanRequest(Base):
         doc="Content of the default observation plan request.",
     )
 
+    filters = sa.Column(
+        psql.JSONB,
+        doc="Filters of the default observation plan request.",
+    )
```
Comment (guynir42) [inline skyportal/models/observation_plan.py:64]: what kind of filters are we talking about here?

File: static/js/components/DefaultObservationPlanTable.jsx [hunk 2]
```diff
@@ -127,25 +127,25 @@ const DefaultObservationPlanTable = ({
...
 
-  const renderEnd_Date = (dataIndex) => {
+  const renderFilters = (dataIndex) => {
     const default_observation_plan = default_observation_plans[dataIndex];
 
     return (
       <div>
         {default_observation_plan
-          ? default_observation_plan.payload.end_date
+          ? default_observation_plan.payload.filters
```
Comment (guynir42) [inline static/js/components/DefaultObservationPlanTable.jsx:148]: You removed the `payload` in line 135 but you use it here? Is that correct?

**Review by guynir42 — APPROVED** [review]
I'm not sure what the filters are. Is that just a dictionary that should replace whatever conditions like start_time?

File: static/js/components/DefaultObservationPlanTable.jsx (hunk 2 above)
Comment (mcoughlin) [reply static/js/components/DefaultObservationPlanTable.jsx:148]: So basically the start_date and end_date don't mean anything in these default plans as they get replaced.

File: skyportal/models/observation_plan.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/models/observation_plan.py]: Adding some extra text here to be more explicit.

File: skyportal/models/observation_plan.py [hunk 3]
```diff
@@ -60,7 +60,7 @@ class DefaultObservationPlanRequest(Base):
 
     filters = sa.Column(
         psql.JSONB,
-        doc="Filters of the default observation plan request.",
+        doc="Filters to determine which of the default observation plan request. get executed for which events",
```
Comment (guynir42) [inline skyportal/models/observation_plan.py:63]:
```suggestion
        doc="Filters to determine which of the default observation plan request get executed for which events",
```

**Pull request merged**